source/user-manual/capabilities/antiflooding.rst

```diff
@@ -194,11 +194,10 @@ When the bucket is in this status, **no events are dropped**.
 Anti-flooding in agent modules
 ------------------------------
 
-In order to avoid agent buffer saturation followed by event loss, the event production rates of Wazuh agent daemons that could cause this saturation have been limited.
+To avoid agent buffer saturation followed by event loss, the event production rates of the Wazuh agent daemons that could cause this saturation has been limited.
```

Comment:
```suggestion
To avoid agent buffer saturation followed by event loss, the event production rates of the Wazuh agent daemons that could cause this saturation have been limited.
```